appengine/go11x/tasks/create_http_task/create_http_task_test.go

```diff
@@ -0,0 +1,47 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"github.com/GoogleCloudPlatform/golang-samples/internal/testutil"
+	"testing"
+)
+
+func TestCreateHttpTask(t *testing.T) {
+	tc := testutil.SystemTest(t)
+	locationID := "us-central1"
+	queueID := "my-appengine-queue"
```

Comment:
Given this is not a queue for appengine, should it still target `my-appengine-queue`? Maybe we should have a second queue to test HTTP Push. Not sure if test concurrency could be a concern, maybe tbp@ has insight there?

Reply:
Queues can handle both app engine and http routing. We can create new queue with a better name if this is confusing.

Reply:
Not sure I understand - does this need to be App Engine? I noticed the `.appspot.com/task_handler` URL below - we probably shouldn't depend on a certain App Engine app being deployed when this test runs. We could deploy it as part of the test, but if it's possible to run the test without that would be even better.

If it's not related to App Engine, could this be in a completely separate directory, say `golang-samples/tasks`?

Reply:
This sample first needed an app engine task handler and the old task queues were tied to app engine hence why it is in the appengine folder. It can be moved if necessary. The url is a placeholder and it doesn't matter for the test to pass. The URL currently set so it mimics the sample url.